src/scenes/ObservationsRouter/org/OrgObservationsListView.tsx

```diff
@@ -190,7 +169,13 @@ export default function OrgObservationsListView({
         columns={columns}
         rows={results}
         orderBy='completedDate'
-        Renderer={OrgObservationsRenderer(theme, activeLocale, goToRescheduleObservation, exportObservation)}
+        Renderer={OrgObservationsRenderer(
+          theme,
+          activeLocale,
+          goToRescheduleObservation,
+          (observationId: number) => exportObservation(observationId, 'csv'),
```

Comment:
Could we pass only one exportObservation function and in the renderer set 'csv' and 'gpx' on the menu items?

Reply:
My thinking was that I want to keep this `csvOrGpx` type private, since it is likely just a one time use.